### PR TITLE
HOTFIX Excluded middlewares on type generation

### PR DESCRIPTION
**Pull request by SavelevMatthew**

### CODE REVIEW

**Review by akarjakin — APPROVED** [review]

File: packages/@core.keystone/test.utils.js [hunk 1]
```diff
@@ -88,15 +88,16 @@ function setFakeClientMode (path) {
     __isAwaiting = true
 }
 
-const prepareKeystoneExpressApp = async (entryPoint, { excludeApps } = {}) => {
+const prepareKeystoneExpressApp = async (entryPoint, { excludeApps, remainingApps } = {}) => {
```
Comment (AntonAL) [inline packages/@core.keystone/test.utils.js:91]: Let's shorten it:

```suggestion
const prepareKeystoneExpressApp = async (entryPoint, { excludeApps, apps: remainingApps } = {}) => {
```

File: bin/create-graphql-schema.js [hunk 2]
```diff
@@ -22,7 +22,9 @@ generates:
 
 
 async function getGraphQLSchema (keystoneModule) {
-    const { keystone } = await prepareKeystoneExpressApp(keystoneModule)
+    const { keystone } = await prepareKeystoneExpressApp(keystoneModule, {
+        remainingApps: ['GraphQLApp'],
```
Comment (AntonAL) [inline bin/create-graphql-schema.js:26]:
```suggestion
        apps: ['GraphQLApp'],
```

**Comment by pahaz** [comment]
It’s better to use excluded apps. You really don’t know about future apps. If some app will change the GQl schema? 

You will hide the error in such cases. I’m add excluded apps especially for that reason rather than remaining apps.

**Comment by pahaz** [comment]
I have use excluded just for performance optimization. Why you need to another one option? It may hide an error

**Pull request closed without merging**